FizzBuzz 1/FizzBuzz/Program.cs

```diff
@@ -7,26 +8,49 @@ class Program
     {
         static void Main(string[] args)
         {
-            for (int i = 1; i <= 100; i++)
+            for (int i = 1; i <= 260; i++)
             {
-                StringBuilder output = new StringBuilder(16);
+                String[] outputArr = new String[5];
 
                 if (i % 3 == 0)
                 {
-                    output.Append("Fizz");
+                    outputArr[0] = "Fizz";
                 }
 
                 if (i % 5 == 0)
                 {
-                    output.Append("Buzz");
+                    outputArr[3] = "Buzz";
                 }
 
-                if (output.Length == 0)
+                if (i % 7 == 0)
                 {
-                    output.Append(i.ToString());
+                    outputArr[4] = "Bang";
                 }
 
-                Console.WriteLine(output);
+                if (i % 11 == 0)
+                {
+                    outputArr = new String[5]; // Tried the Initialize() method but wasn't working
```

Comment:
If you have comments like this you can leave a comment on your own pull request, that way you don't need to add comments to the code that you later need to remember to remove 🙂